flatliners/dfgen.py

```diff
@@ -0,0 +1,67 @@
+from .baseflatliner import BaseFlatliner
+import pandas as pd
+from datetime import datetime
+
+
+class DfGenerator(BaseFlatliner):
```

Comment:
Could you use a more descriptive class name?
Like `AlertsCorrelation`? Not so sure what you want to do in this class

Reply:
I was starting from basic, like making a dataframe and taking snapshot for a window. I will correct these names in next pr